buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -1589,9 +1589,9 @@ def download_boot_jdk(bootJDKVersion, bootJDK) {
                 gzip -cd "\$sdkFile" | tar xof - -C ${bootJDK} --strip=${dirStrip}
             fi
             ${bootJDK}/bin/java -version
-            rm -f "\$sdkFile"
         """
     }
+    cleanWs disableDeferredWipeout: true, deleteDirs: true
```

Comment:
Wouldn't it fail intermittently on windows? Could use `cleanWorkspace(false)`?

Reply:
Done